ticdc/ticdc-new-arch.md

```diff
@@ -0,0 +1,160 @@
+| **任务触发机制**          | 定时器触发的大循环，每隔 50 ms 检查任务，处理性能有限    | 由事件驱动，包括 DML、DDL 变更及 Changefeed 操作，队列中的事件会被尽快处理，无需等待 50 ms 的固定间隔，从而减少了额外的延迟 |
+| **任务调度方式**          | 每个 Changefeed 运行一个主循环，轮询检查任务 | 事件被放入队列后，由多个线程并发消费处理   |
+| **任务处理效率**          | 每个任务需要经过多个周期，存在性能瓶颈    | 事件可以立即处理，无需等待固定间隔，减少延迟 |
+| **资源消耗**              | 频繁检查不活跃表，浪费 CPU 资源    | 消费线程仅处理队列中的事件，无需检查不活跃任务 |
+| **复杂度**                | O(n)，表数量增多时性能下降               | O(1)，不受表数量影响，效率更高           |
+| **CPU 利用率**            | 每个 Changefeed 只能利用一个逻辑 CPU     | 能充分利用多核 CPU 的并行处理能力        |
+| **扩展能力**              | 受限于 CPU 数量，扩展性差                | 通过多线程消费和事件队列，可扩展性强     |
+| **Changefeed 干扰问题**   | 中央控制节点 (Owner) 会造成 Changefeed 之间的干扰 | 事件驱动模式避免了 Changefeed 之间的干扰 |
+
+![TiCDC 新老架构对比](/media/ticdc/ticdc-new-arch-2.png)
+
+## 使用限制
+
+作为实验性特性，TiCDC 新架构尚未完全实现旧架构中的所有功能，这些功能将在后续的 GA 版本中完整实现，具体包括：
+
+- [拆分 Update 事件](/ticdc/ticdc-split-update-behavior.md)
+- [灾难场景的最终一致性复制](/ticdc/ticdc-sink-to-mysql.md#灾难场景的最终一致性复制)
+- 拆分大事务
+- [TiCDC Avro Protocol](/ticdc/ticdc-avro-protocol.md)
+- [TiCDC CSV Protocol](/ticdc/ticdc-csv.md)
+- [TiCDC Debezium Protocol](/ticdc/ticdc-debezium.md)
+- [TiCDC Simple Protocol](/ticdc/ticdc-simple-protocol.md)
+- [Event Filter 事件过滤器](/ticdc/ticdc-filter.md#event-filter-事件过滤器-从-v620-版本开始引入)
+- [TiCDC 单行数据正确性校验](/ticdc/ticdc-integrity-check.md)
+- [TiCDC 双向复制](/ticdc/ticdc-bidirectional-replication.md)
+- [同步数据到 Pulsar](/ticdc/ticdc-sink-to-pulsar.md)
+- [同步数据到存储服务](/ticdc/ticdc-sink-to-cloud-storage.md)
+
+## 部署指南
+
+TiCDC 新架构仅支持 v7.5.0 或者以上版本的 TiDB 集群，使用之前需要确保 TiDB 集群版本满足要求。
+
+你可以选择以下部署方式之一：
+
+- [使用 TiUP 部署启用新架构 TiCDC 的全新 TiDB 集群](#使用-tiup-在原有-tidb-集群中部署启用新架构的-ticdc-组件)
+- [使用 TiUP 在原有 TiDB 集群中部署启用新架构的 TiCDC 组件](#使用-tiup-部署启用新架构-ticdc-的全新-tidb-集群)
+
+### 使用 TiUP 部署启用新架构 TiCDC 的全新 TiDB 集群
+
+在使用 TiUP 部署 v9.0.0 或者以上版本的全新 TiDB 集群时，支持同时部署启用新架构的 TiCDC 组件。你需要在 TiUP 启动 TiDB 集群时的配置文件中加入 TiCDC 组件相关的部分并启用新架构，以下是一个示例：
+
+```yaml
+cdc_servers:
+  - host: 10.0.1.20
+    config:
+      newarch: true
+  - host: 10.0.1.21
+    config:
+      newarch: true
+```
+
+> **注意：**
+>
+> `newarch` 配置项仅用于 TiCDC 新架构，不添加 `newarch` 配置项则默认使用老架构。在使用 TiCDC 老架构时，请勿在配置文件中添加 `newarch` 配置项，否则可能会导致解析失败。
+
+更多详细操作，请参考[使用 TiUP 部署包含 TiCDC 组件的全新 TiDB 集群](/ticdc/deploy-ticdc.md#使用-tiup-部署包含-ticdc-组件的全新-tidb-集群)。
+
+### 使用 TiUP 在原有 TiDB 集群中部署启用新架构的 TiCDC 组件
+
+1. 如果你的 TiDB 集群中尚无 TiCDC 节点，参考[扩容 TiCDC 节点](/scale-tidb-using-tiup.md#扩容-ticdc-节点)在集群中扩容新的 TiCDC 节点，否则跳过该步骤。
+
+2. 如果你的 TiDB 集群为 v9.0.0 之前版本，需要按照以下方式手动下载 v9.0.0 或以上版本的 TiCDC 离线包，并将下载的 TiCDC 二进制文件 Patch 到你的 TiDB 集群，否则跳过该步骤。
+
+    > **警告：**
+    >
+    > 升级至 TiCDC 新架构后，无法回退至旧架构。
+
+    1. 下载 v9.0.0 或以上版本的 TiCDC 二进制文件。
+
+        该文件下载链接格式为 `https://tiup-mirrors.pingcap.com/cdc-${version}-${os}-${arch}.tar.gz`。其中，`${version}` 为 TiCDC 版本号，`${os}` 为你的操作系统，`${arch}` 为组件运行的平台（`amd64` 或 `arm64`）。
+
+        例如，可以使用以下命令下载 Linux 系统 x86-64 架构的 TiCDC v9.0.0 的二进制文件：
+
+       ```shell
+       wget https://tiup-mirrors.pingcap.com/cdc-v9.0.0-linux-amd64.tar.gz
```

Comment:
This link does not work for now. As this doc will be displayed only in the dev version, is it okay?

Reply:
All v9.0.0 in this doc should be replaced with the truly released version(i.e. v9.0.0-beta.1), so the real link should be something like `https://tiup-mirrors.pingcap.com/cdc-v9.0.0-beta.1-linux-amd64.tar.gz`